justine.py

```diff
@@ -0,0 +1,30 @@
+def justine_delima():
+    user_choice = 0
+
+    while user_choice != 4:
+        print("Nice to know you! I am Justine Delima")
+        print('[1] Basic Information')
+        print('[2] Goals')
+        print('[3] Comments')
+        print('[4] Exit')
+        user_choice = int(input("Enter your choice: "))
+
+        match user_choice:
+            case 1: #  Basic Info
+                print('\nAge: 19')
+                print('Birthdate: February 24, 2005')
+                print('Currentt School Attended: Polytechnic of the Philippines-Taguig\n')
```

Comment:
Remove extra 't' to "Current"

Reply:
Fixed in commit 2a2726a90153b2b1104b360eaa45b4868e5556fb